### ISSUE TITLE
Image can't be placed on PaperMC 1.17.1 (Unable to get Splatter Map attribute on item)

### ISSUE THREAD

**Issue by mk499490** [zDevelopers/ImageOnMap#216]
When I place the map, the first map tile(upper-left one) can be placed. But other maps can't be placed.

### My environment:
**Operating system:** Ubuntu Server 20.04 LTS 64bit
**MC server version:** PaperMC 1.17.1 266 (I specify "-Djava.awt.headless=true" in my startup script)
**ImageOnMap version:** 4.1.2
**Java version:** OpenJDK 16.0.1

### Stacktrace:
```
[04:42:06 ERROR]: [ImageOnMap] Unable to get Splatter Map attribute on item
fr.zcraft.imageonmap.quartzlib.tools.reflection.NMSException: Unable to find class: ItemStack
        at fr.zcraft.imageonmap.quartzlib.components.nbt.NBT.getMinecraftClass(NBT.java:248) ~[ImageOnMap-4.1.2.jar:?]
        at fr.zcraft.imageonmap.quartzlib.components.nbt.NBT.init(NBT.java:265) ~[ImageOnMap-4.1.2.jar:?]
        at fr.zcraft.imageonmap.quartzlib.components.nbt.NBT.fromItemStack(NBT.java:72) ~[ImageOnMap-4.1.2.jar:?]
        at fr.moribus.imageonmap.ui.SplatterMapManager.hasSplatterAttributes(SplatterMapManager.java:155) ~[ImageOnMap-4.1.2.jar:?]
        at fr.moribus.imageonmap.ui.MapItemManager.onItemFramePlace(MapItemManager.java:255) ~[ImageOnMap-4.1.2.jar:?]
        at fr.moribus.imageonmap.ui.MapItemManager.onEntityInteract(MapItemManager.java:349) ~[ImageOnMap-4.1.2.jar:?]
        at com.destroystokyo.paper.event.executor.StaticMethodHandleEventExecutor.execute(StaticMethodHandleEventExecutor.java:38) ~[patched_1.17.1.jar:git-Paper-266]
        at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[patched_1.17.1.jar:git-Paper-266]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[patched_1.17.1.jar:git-Paper-266]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:624) ~[patched_1.17.1.jar:git-Paper-266]
        at net.minecraft.server.network.PlayerConnection$5.a(PlayerConnection.java:2422) ~[patched_1.17.1.jar:git-Paper-266]
        at net.minecraft.server.network.PlayerConnection$5.a(PlayerConnection.java:2472) ~[patched_1.17.1.jar:git-Paper-266]
        at net.minecraft.network.protocol.game.PacketPlayInUseEntity$d.a(PacketPlayInUseEntity.java:131) ~[patched_1.17.1.jar:git-Paper-266]
        at net.minecraft.network.protocol.game.PacketPlayInUseEntity.a(PacketPlayInUseEntity.java:80) ~[patched_1.17.1.jar:git-Paper-266]
        at net.minecraft.server.network.PlayerConnection.a(PlayerConnection.java:2414) ~[patched_1.17.1.jar:git-Paper-266]
        at net.minecraft.network.protocol.game.PacketPlayInUseEntity.a(PacketPlayInUseEntity.java:67) ~[patched_1.17.1.jar:git-Paper-266]
        at net.minecraft.network.protocol.game.PacketPlayInUseEntity.a(PacketPlayInUseEntity.java:12) ~[patched_1.17.1.jar:git-Paper-266]
        at net.minecraft.network.protocol.PlayerConnectionUtils.lambda$ensureRunningOnSameThread$1(PlayerConnectionUtils.java:56) ~[patched_1.17.1.jar:git-Paper-266]
        at net.minecraft.server.TickTask.run(TickTask.java:18) ~[patched_1.17.1.jar:git-Paper-266]
        at net.minecraft.util.thread.IAsyncTaskHandler.executeTask(IAsyncTaskHandler.java:149) ~[patched_1.17.1.jar:git-Paper-266]
        at net.minecraft.util.thread.IAsyncTaskHandlerReentrant.executeTask(IAsyncTaskHandlerReentrant.java:23) ~[patched_1.17.1.jar:git-Paper-266]
        at net.minecraft.server.MinecraftServer.executeTask(MinecraftServer.java:1418) ~[patched_1.17.1.jar:git-Paper-266]
        at net.minecraft.server.MinecraftServer.executeTask(MinecraftServer.java:192) ~[patched_1.17.1.jar:git-Paper-266]
        at net.minecraft.util.thread.IAsyncTaskHandler.executeNext(IAsyncTaskHandler.java:122) ~[patched_1.17.1.jar:git-Paper-266]
        at net.minecraft.server.MinecraftServer.bf(MinecraftServer.java:1396) ~[patched_1.17.1.jar:git-Paper-266]
        at net.minecraft.server.MinecraftServer.executeNext(MinecraftServer.java:1389) ~[patched_1.17.1.jar:git-Paper-266]
        at net.minecraft.util.thread.IAsyncTaskHandler.awaitTasks(IAsyncTaskHandler.java:132) ~[patched_1.17.1.jar:git-Paper-266]
        at net.minecraft.server.MinecraftServer.sleepForTick(MinecraftServer.java:1367) ~[patched_1.17.1.jar:git-Paper-266]
        at net.minecraft.server.MinecraftServer.x(MinecraftServer.java:1278) ~[patched_1.17.1.jar:git-Paper-266]
        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:319) ~[patched_1.17.1.jar:git-Paper-266]
        at java.lang.Thread.run(Thread.java:831) [?:?]
Caused by: java.lang.ClassNotFoundException: net.minecraft.server.v1_17_R1.ItemStack
        at org.bukkit.plugin.java.PluginClassLoader.loadClass0(PluginClassLoader.java:146) ~[patched_1.17.1.jar:git-Paper-266]
        at org.bukkit.plugin.java.PluginClassLoader.loadClass(PluginClassLoader.java:103) ~[patched_1.17.1.jar:git-Paper-266]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:519) ~[?:?]
        at java.lang.Class.forName0(Native Method) ~[?:?]
        at java.lang.Class.forName(Class.java:375) ~[?:?]
        at fr.zcraft.imageonmap.quartzlib.tools.reflection.Reflection.getMinecraftClassByName(Reflection.java:106) ~[ImageOnMap-4.1.2.jar:?]
        at fr.zcraft.imageonmap.quartzlib.components.nbt.NBT.getMinecraftClass(NBT.java:246) ~[ImageOnMap-4.1.2.jar:?]
        ... 30 more
```

**Comment by srnyx** [comment]
https://github.com/zDevelopers/ImageOnMap/issues/198

**Comment by RMHavelaar** [comment]
I am having the exact came issue.

**Comment by srnyx** [comment]
> I am having the exact came issue.

https://github.com/zDevelopers/ImageOnMap/issues/198

**Comment by Vlammar** [comment]
This is fixed with the 4.2.0 https://github.com/zDevelopers/ImageOnMap/releases/tag/v4.2.0 . Have a nice 4.2 :)
